setup.pl

```diff
@@ -465,7 +467,7 @@
 		$host = &get_system_hostname();
 		$cert = &tempname();
 		$key = &tempname();
-		open(SSL, "| openssl req -newkey rsa:512 -x509 -nodes -out $cert -keyout $key -days 1825 >/dev/null 2>&1");
+		open(SSL, "| openssl req -newkey rsa:2048 -x509 -nodes -out $cert -keyout $key -days 1825 -sha256 >/dev/null 2>&1");
```

Comment:
N: Users reported it in the past, that they couldn't start Webmin with `miniserv.pem` generated with 512 key length, as 512 key length is forbidden nowadays and it will always fail with an error upon starting Webmin, saying something like `cannot create security context`.

I think in 2022 we have to switch to 4096 to be default key length. What do you think?

Reply:
I agree, but that change shouldn't be in this PR as it's unrelated.

Reply:
okay, i'll remove it and commit that directly.